Resources/Prototypes/Entities/Structures/Doors/Turnstiles/turnstiles.yml

```diff
@@ -0,0 +1,126 @@
+﻿- type: entity
+  id: Turnstile
+  parent: BaseStructure
+  name: turnstile
+  description: One-way access - control the flow of foot traffic.
+  components:
+  - type: MeleeSound
+    soundGroups:
+      Brute:
+        path:
+          "/Audio/Weapons/smash.ogg"
+  - type: InteractionOutline
```

Comment:
probably inherited from structure

Reply:
BaseStructure doesn't seem to have InteractionOutline.

Reply:
ok then its fine